okhttp/src/main/java/com/squareup/okhttp/MediaType.java

```diff
@@ -31,6 +31,64 @@
   private static final Pattern PARAMETER = Pattern.compile(
       ";\\s*(?:" + TOKEN + "=(?:" + TOKEN + "|" + QUOTED + "))?");
 
+  /**
+   * A {@link MediaType} constant representing the wildcard media type.
+   */
+  public static final MediaType WILDCARD = new MediaType("*/*", "*", "*", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xml" media type.
+   */
+  public static final MediaType APPLICATION_XML =
+                  new MediaType("application/xml", "application", "xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/atom+xml" media type.
+   */
+  public static final MediaType APPLICATION_ATOM_XML =
```

Comment:
Drop this? I'd like to limit this to a very small set of very popular media types:

XML,
JSON,
FORM_URLENCODED,
FORM_DATA,
OCTET_STREAM,
and TEXT_PLAIN

Do we need two different media types for XML? That's unfortunate!